AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAdminForUserUnenrolls.cs

```diff
@@ -22,58 +24,93 @@ public NotifyAdminForUserUnenrolls(IMediator mediator, IOptions<GeneralSettings>
+                    message.AppendLine($"   Task Start Date: {taskModel.StartDateTime?.Date.ToShortDateString()}");
                 }
-            }
+                message.AppendLine($"   Remaining/Required Volunteers: {remainingRequiredVolunteersPhrase}");
+                message.AppendLine();
 
-            var command = new NotifyVolunteersCommand
-            {
-                ViewModel = new NotifyVolunteersViewModel
+
+                if (isActivityUnenroll)
                 {
-                    EmailMessage = message.ToString(),
-                    HtmlMessage = message.ToString(),
-                    EmailRecipients = new List<string> { campaignContact.Contact.Email },
-                    Subject = subject
+                    var assignedTasks = notificationModel.Tasks.Where(t => t.AssignedVolunteers.Any(au => au.UserId == notificationModel.Volunteer.Id)).ToList();
+                    if (assignedTasks.Count == 0)
+                    {
+                        message.AppendLine("This volunteer had not been assigned to any tasks.");
+                    }
+                    else
+                    {
+                        message.AppendLine("This volunteer had been assigned to the following tasks:");
+                        message.AppendLine("   Name             Description               Start Date           TaskLink");
+                        foreach (var assignedTask in assignedTasks)
+                        {
+                            var taskLink = $"View task: {_options.Value.SiteBaseUrl}Admin/Task/Details/{assignedTask.Id}";
+                            message.AppendFormat("   {0}{1}{2:d}{3}",
+                                assignedTask.Name?.Substring(0, Math.Min(15, assignedTask.Name.Length)).PadRight(17, ' ') ??
+                                "None".PadRight(17, ' '),
+                                assignedTask.Description?.Substring(0, Math.Min(25, assignedTask.Description.Length)).PadRight(26, ' ') ??
+                                "None".PadRight(26, ' '),
+                                assignedTask.StartDateTime?.Date.ToShortDateString().PadRight(21, ' ') ?? "".PadRight(21, ' '),
+                                taskLink);
+                        }
+                    }
                 }
-            };
+
+                
+                var command = new NotifyVolunteersCommand
+                {
+                    ViewModel = new NotifyVolunteersViewModel
+                    {
+                        EmailMessage = message.ToString(),
+                        HtmlMessage = message.ToString(),
+                        EmailRecipients = new List<string> { campaignContact.Contact.Email },
+                        Subject = subject
+                    }
+                };
             await _mediator.SendAsync(command).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                // ignored
```

Comment:
I see what you're doing here, but this makes it hard to troubleshoot :(

Can you perhaps inject something like an `ILogger<NotifyAdminForUserUnenrolls> logger` into the constructor and log out the exception? Logging to the console is configured for the project, so at least if we had a debugger attached (or we were checking the Azure logs) we'd see the problem with notifications.